tpl/img_optm/settings.tpl.php

```diff
@@ -107,17 +108,13 @@
 			<?php $this->build_switch( $id ); ?>
 			<div class="litespeed-desc">
 				<?php echo __( 'Request WebP versions of original images when doing optimization.', 'litespeed-cache' ); ?>
+				<?php echo sprintf( __( 'Significantly improve load time by replacing images with their optimized %s versions.', 'litespeed-cache' ), '.webp' ); ?>
+				<br /><?php Doc::notice_htaccess(); ?>
+				<br /><?php Doc::crawler_affected(); ?>
 			</div>
 		</td>
 	</tr>
-
-	<?php
-		if ( ! is_multisite() ) :
-			// webp
-			require LSCWP_DIR . 'tpl/img_optm/settings.media_webp.tpl.php';
```

Comment:
The reason we use separate file is for network settings as well. If you remove this file, in `img_optm/network_settings.tpl.php`, there will be no this option

Reply:
Thanks, missed that. Should be fixed now.